Vagrantfile

```diff
@@ -137,8 +137,8 @@ def configureLinuxProvisioners(vmCfg)
 		path: './scripts/vagrant-linux-priv-rkt.sh'
 
 	vmCfg.vm.provision "shell",
-		privileged: true,
-		path: './scripts/vagrant-linux-priv-ui.sh'
+		privileged: false,
+		path: './scripts/vagrant-linux-unpriv-ui.sh'
```

Comment:
You're going to need to actually rename the scripts if you're going to reference it by this new name.

Reply:
it's there 😄

Reply:
![screen shot 2018-10-05 at 3 50 38 pm](https://user-images.githubusercontent.com/1675087/46556999-6c37cf00-c8b6-11e8-99da-8b98d1ab8922.png)

Reply:
Doh!